tests/framework/clients/topic.go

```diff
@@ -62,3 +63,48 @@ func (t *TopicOperation) CheckTopic(topicName string) bool {
 	logger.Infof("topic ARN is %q", topicARN)
 	return true
 }
+
+func (t *TopicOperation) CreateHTTPServer() error {
```

Comment:
I need to change this function to make more generic, will address in next commit